src/inference/src/dev/threading/cpu_streams_executor.cpp

```diff
@@ -133,9 +133,13 @@ struct CPUStreamsExecutor::Impl {
                             ? (small_core ? small_core_offset : (logic_core ? 0 : 1))
                             : 0;
 
+#    ifdef _WIN32
                     _taskArena.reset(new custom::task_arena{custom::task_arena::constraints{}
                                                                 .set_core_type(selected_core_type)
                                                                 .set_max_concurrency(max_concurrency)});
+#    else
+                    _taskArena.reset(new custom::task_arena{max_concurrency});
```

Comment:
why core types are not used for Linux?

Reply:
For linux, core binding is implemented in the follow-up code,  so here core types are not used for Linux.

Reply:
@ilya-lavrenov For Linux, we will use OV observer in CPU_StreamsExecutor to handle core types.